.restyled.yaml

```diff
@@ -178,26 +178,27 @@ restylers_version: "20191031"
 #   The command (and any required argument(s)) to perform the Restyling.
 #
 restylers:
-image: restyled/restyler-clang-format:v9.0.0
-command:
-- clang-format
-- "-i"
-arguments: []
-include:
-- "**/*.c"
-- "**/*.cc"
-- "**/*.cpp"
-- "**/*.cxx"
-- "**/*.c++"
-- "**/*.C"
-- "**/*.cs"
-- "**/*.h"
-- "**/*.hh"
-- "**/*.hpp"
-- "**/*.hxx"
-- "**/*.h++"
-- "**/*.H"
-- "**/*.java"
-- "**/*.js"
-- "**/*.m"
-interpreters: []
+  - name: prettier
```

Comment:
name should be [clang-format](https://github.com/restyled-io/restyled.io/wiki/Available-Restylers#clang-format) not [prettier](https://github.com/restyled-io/restyled.io/wiki/Available-Restylers#prettier).
This probably works to, but uses the prettier javascript restyler but changes all it's default settings to the default settings of the clang-format restyler.

Reply:
fixed in commit https://github.com/GSI-CS-CO/chart-fx/pull/69/commits/b7f80b4cef954ca52696fa2487eb9465e98bdd0f